src/utils/src/arg_parser.rs

```diff
@@ -476,7 +488,16 @@ mod tests {
         assert!(arguments.parse(&args).is_ok());
         assert!(arguments.args.contains_key("help"));
 
-        let arg_parser = build_arg_parser();
+        let mut arguments = arg_parser.arguments().clone();
```

Comment:
Little nit: no need here for the `let mut` part.

Reply:
done. thanks. ;)